homeassistant/components/miele/sensor.py

```diff
@@ -144,7 +144,6 @@ class MieleSensorDefinition:
             MieleAppliance.STEAM_OVEN,
             MieleAppliance.MICROWAVE,
             MieleAppliance.COFFEE_SYSTEM,
-            MieleAppliance.ROBOT_VACUUM_CLEANER,
```

Comment:
Let's merge this one during beta so we don't have to deprecate it

Reply:
Please elaborate. Do you mean that the program phase sensor should be left in place for the time being?
Or do you mean that the vacuum platform should be merged into beta?

Reply:
I meant hat we should remove the program phase sensor from the beta, as when we release this, we would have to deprecate it for 6 months using issues